ui/src/components/Employees/style.module.scss

```diff
@@ -0,0 +1,101 @@
+@import 'src/scss/lib';
+
+.employee-list-container {
+  color: $color-text-dim2;
+  padding: 20px;
+  text-align: left;
+  width: 55%;
+
+  @media(max-width: $md) {
+    margin-top: 100px;
+  }
+
+  @media(max-width: $sm) {
+    margin-top: 60px;
+  }
+}
+
+.employee-content {
+  background: $color-select-active;
+  border-radius: 5px;
+  display: grid;
+  grid-auto-rows: minmax(2em, auto);
+  grid-column-gap: 1em;
+  grid-row-gap: 0;
+  grid-template-columns: 20% 25% 1fr;
+  height: 6em;
+  margin-top: 2em;
+  padding: 15px;
+}
+
+.list-section1 {
```

Comment:
Please justify your choice of name here. As an outsider, `list-section1` is revealing very little to me about what this class is about.

Reply:
I am resolving all CSS related issues since we are going to change CSS anyway. I've created issues for implementing new styles.